components/abstractions/src/main/java/com/microsoft/kiota/ApiExceptionBuilder.java

```diff
@@ -0,0 +1,51 @@
+package com.microsoft.kiota;
+
+import jakarta.annotation.Nonnull;
+
+import java.util.Objects;
+
+/** Builder class for ApiException. */
+public class ApiExceptionBuilder {
+
+    private ApiException value = null;
+    public ApiExceptionBuilder() {
+    }
+    public ApiExceptionBuilder(ApiException base) {
```

Comment:
all methods in this class are missing doc comments

Reply:
done